kwalitee/repo.py

```diff
@@ -92,6 +94,32 @@ def get_project_type(self):
             type = 'python' if any(Path(x).is_file() for x in ['setup.py', 'requirements.txt', 'requirements_test.txt']) else 'bashlib'
         return type
 
+    def get_dependencies(self):
+        f = open('deps.json')
+        deps_file = json.load(f)
+
+        return deps_file[self.id]
+
+
+    def get_dependency_conflicts(self):
+        f = open('dep_conflicts.json')
+        json_file = json.load(f)
+
+        result = {}
+        for pkg in json_file:
+            if self.id in json_file[pkg].keys():
```

Comment:
```suggestion
            if self.id in json_file[pkg]:
```

that's what the `in` check does for dictionaries by default.